src/domain/community/userAdmin/tabs/UserAdminNotificationsPage.tsx

```
@@ -1,12 +1,17 @@
-import { useMemo } from 'react';
-import { useUpdateUserSettingsMutation, useUserSettingsQuery } from '@/core/apollo/generated/apollo-hooks';
+import { useCallback, useMemo, useReducer } from 'react';
```

Biome (2.4.7)

[error] Several of these imports are unused.

(lint/correctness/noUnusedImports)